CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,35 @@
 
 import Foundation
 
-print("Hello, World!")
+func printIceCreamPart(width: Int = 7, height: Int = 9) {
+    for _ in 1...height {
+        for _ in 1...width {
+            print("*", terminator: "")
+        }
+        print("")
+    }
+}
 
+func printIceCreamStickPart(iceCreamWidth: Int = 7, length: Int = 3) {
+    let iceCreamWidthHalf = iceCreamWidth/2
```

Comment:
```suggestion
    let iceCreamWidthHalf = iceCreamWidth / 2
```
이것은 개인의 취향일수도 있지만 가독성을 위해 연산자에 띄어쓰기를 넣어주면 좋을 거 같습니다!

Reply:
앗! 맞네요. 구현하는데 정신이 팔려서 캐치를 못했네요ㅋㅋ 감사합니다.